README.md

```diff
@@ -77,16 +77,13 @@ Updated 0 as deleted
 # prepare to wait a long time
 ```
 
-Packaging
+Packaging and release
 ---------
 
 For maintainers:
 
-1. Update the `__version__` in setup.py.
-2. Build, check and upload to PyPI as follows:
+1. Update the `__version__` in setup.py. Commit. Push.
+2. Create a release on GitHub for this version.
+3. When published, a [GitHub action](https://github.com/matrix-org/synapse-s3-storage-provider/runs/4312656424?check_suite_focus=true) will build the package and upload to [PyPI](https://pypi.org/project/synapse-s3-storage-provider/).
 
-```shell
-python -m build
-twine check dist/*
-twine upload dist/*
-```
+**NB.** `matrix.org` is currently configured to `git checkout` this module directly and does NOT pull from PyPI.
```

Comment:
Not sure if we should write stuff about matrix.org ops in readmes, but maybe that's just me being too paranoid.

Reply:
Agree on this, this doesn't seem like the right place for details about the matrix.org deployment

Reply:
Thanks, will revert this bit.